cloudflare-d1/vite.config.ts

```diff
@@ -37,14 +37,12 @@ export default defineConfig(({ isSsrBuild }) => ({
     },
   },
   plugins: [
-    vitePluginViteNodeMiniflare({
-      entry: "./workers/app.ts",
-      miniflareOptions: (options) => {
-        options.compatibilityDate = "2024-11-18";
-        options.compatibilityFlags = ["nodejs_compat"];
-        options.d1Databases = { DB: "your-database-id" };
-        // match where wrangler applies migrations to
-        options.d1Persist = ".wrangler/state/v3/d1";
+    cloudflareDevProxy({
+      getLoadContext({ context }) {
+        return getLoadContext(
+          // @ts-expect-error - context types are unaware of the proxied values
+          context.cloudflare
```

Comment:
I could not figure out a way to avoid this mismatch of types and get `cloudflareDevProxy` to use the right `context` type for `getLoadContext`. If anyone has a solution I would very much appreciate a contribution